reactos/drivers/usb/usbhub_new/dbg_uhub.h

```diff
@@ -0,0 +1,5 @@
+#ifndef DBG_UHUB_H__
+#define DBG_UHUB_H__
+
+
+#endif /* DBG_UHUB_H__ */
```

Comment:
I wanted to use it like usbdebug.h in usbport (for splitting DPRINTs into groups). But while this is not used and can be deleted.